Sources/HipMobileUI/Views/ExhibitPreviewView.xaml

```diff
@@ -41,26 +41,26 @@
 
                     <StackLayout x:Name="QuestionView" Grid.Row="2" Padding="10,0"
                                  HorizontalOptions="CenterAndExpand" VerticalOptions="CenterAndExpand">
-                        <Label Text="{Binding Question}" FontSize="Medium" HorizontalTextAlignment="Start" />
+                        <Label Text="{Binding Question}" FontSize="Medium" HorizontalTextAlignment="Start" Style="{DynamicResource Ssp-Regular}"/>
                     </StackLayout>
 
                     <StackLayout x:Name="ButtonView" Grid.Row="2" Orientation="Vertical" VerticalOptions="End" Padding="0">
                         <BoxView HeightRequest="1" Color="{StaticResource DarkGrayColor}" />
                         <StackLayout Orientation="Horizontal" Padding="12,0,0,0"
                                      BackgroundColor="{StaticResource LightGrayColor}" VerticalOptions="End">
                             <StackLayout Orientation="Horizontal" HorizontalOptions="FillAndExpand">
-                                <Label Text="{helpers:Translate Yes}" FontSize="20"
-                                       TextColor="{StaticResource AccentColor}"
+                                <Label Text="{helpers:Translate ExhibitOrRouteNearby_Confirm}" FontSize="20"
```

Comment:
Like above the strings `Yes` and `No` are the official version that should be used now instead of the long names.